src/main/webapp/dataset-license-terms.xhtml

```diff
@@ -52,9 +52,16 @@
                                </ui:fragment>
                                <ui:fragment rendered="#{!empty editMode}">
                                    <p class="help-block"><h:outputText value="#{bundle['file.dataFilesTab.terms.list.license.edit.description']}" escape="false"/></p>
+
                                    <p:selectOneMenu id="licenses" value="#{termsOfUseAndAccess.license}" style="width: 40%" converter="licenseConverter">
-                                       <f:selectItems value="#{licenseServiceBean.listAllActive()}"
+                                       <c:if test="#{!datasetPage}">
```

Comment:
I'm a little confused why this is here. Don't we always select licenses from the dataset page?

Reply:
Licenses can also be set on the dataset template.